help/en/package/jmri/jmrit/withrottle/Protocol.shtml

```diff
@@ -59,13 +59,15 @@
       number of lines of text that provide information about the
       JMRI instance to which you've connected. Your application
       then sends commands to JMRI. Each of these commands is plain
-      text, with a newline character at the end (0A hex). Similarly,
-      lines sent back from JMRI are terminated by a newline.  Other
-      implementations of the WiThrottle protocol will terminate the command
-      with a carriage return (0D hex) followed by a newline.</p>
+      text, with a system-specific newline character at the end (typically 0A hex). Similarly,
```

Comment:
System specific new line character does not make sense here.

The newline character is a specific ASCII character.  If you are going to change this ( and I strongly suggest it doesn’t change) it should be system specific end of line character.

Reply:
> System specific new line character does not make sense here.
> 
> The newline character is a specific ASCII character. 

And I learned that "line feed" is a specific ASCII character and "newline" is synonymous with "end of line".

> If you are going to change this ( and I strongly suggest it doesn’t change) it should be system specific end of line character.

I'm not sure what you mean by "this". Do you mean do not change the documentation or do not change the protocol?

Reply:
I was referring to the documentation of the protocol.

My preference would be to change the protocol as implemented in JMRI so that it is consistent.

As I stated in my reply to #6744 we should be conservative in what we send out.  Sending out different characters based on the operating system is not the conservative approach.